libdef.d.ts

```diff
@@ -0,0 +1,15 @@
+// @strict-assignment
```

Comment:
Removing this line results in code in test.ts using these declarations with strict-assignment checking even if when using the `--strict-assignment` flag.